src/com/intellij/idea/plugin/hybris/settings/HybrisApplicationSettingsForm.java

```diff
@@ -57,6 +57,8 @@
     private JCheckBox defaultPlatformInReadOnly;
     private JTextField groupPlatformTextField;
     private JCheckBox followSymlink;
+    private JCheckBox withMavenSources;
```

Comment:
Could we add these checkboxes to the project import wizard?

Reply:
Done